rpm/bundle-rust-npm.py

```diff
@@ -263,8 +266,8 @@ def write_provides_bundled(provides_lines: List[str], spec_file: str, cleaned: b
 
     license_string = create_license_line(rust_crates, npm_packages)
     replace_license(args.spec_file, license_string)
-    log.info(f"Specfile {args.spec_file} is successfully modified! Please:\n"
-              "1. Open the specfile with your editor of choice\n"
-              "2. Make sure that Provides with bundled crates are correct\n"
-              "3. Follow the instructions for 'License:' field and remove the helper comments")
+    print(f"Spec file {args.spec_file} is successfully modified! Please:")
+    print("1. Open the spec file with your editor of choice")
+    print("2. Make sure that 'Provides:' with bundled crates are correct")
+    print("3. Follow the instructions for 'License:' field and remove the helper comments")
```

Comment:
Minor thing - we probably don't need this las line when we do a `fix-it` run.

Reply:
Yes, I agree. I have another series of changes coming up, one of them implements this :)

Reply:
@droideck, sorry for the delay, I pushed another series of commits, please take a look.